js/script.js

```diff
@@ -0,0 +1,6 @@
+const toggleMenu = () => {
+  document.querySelector('.mobile-nav').classList.toggle('block');
+};
+
+document.querySelectorAll('#mobile-nav a').forEach((item) => item.addEventListener('click', toggleMenu));
```

Comment:
You should replace `#mobile-nav` with `.mobile-nav` as shown:

`document.querySelectorAll('.mobile-nav a').forEach((item) => item.addEventListener('click', toggleMenu));`

Since `mobile-nav` is a class, it should be prefixed with a `.` only ids are prefixed with `#`
**Introducing this change would also fix the mobile menu not closing by tapping a link.**